client/my-sites/pages/page.jsx

```diff
@@ -134,6 +138,10 @@ const Page = React.createClass( {
 			return null;
 		}
 
+		if ( this.props.hasStaticFrontPage && this.props.isPostsPage ) {
```

Comment:
Do we need to check both of these? Or is `this.props.isPostsPage` enough? I would think it is, or are there cases where a page can be set as the posts page without a static front page being set? If concerned about the two settings being out of sync, I would say to move the check for static front page into the `isPostsPage` selector itself.